src/components/accounts/index.tsx

```diff
@@ -83,18 +91,21 @@ interface AccountsProps {
 
 export const Accounts = ({ identities, showAddAccount, next }: AccountsProps) => {
   const dispatch = useDispatch();
+  const [loading, setLoading] = useState(false);
   return (
-    <Flex flexDirection="column">
+    <Flex flexDirection="column" position="relative">
```

Comment:
I think probs we need to have position=relative on the account item itself, not the parent wrapper. 

What is the expected behavior of having multiple IDs? I am assuming we want the ID that we choose to grey out, and have the loading within that account item. Can you test with multiple IDs?

Reply:
Changed to flex, word wraps